### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by harshdamaniahd** [pnp/powershell#2453] [Q&A]
I am trying to update modified by and modifed field, it works till this stage
`$result = Set-PnPListItem -List "Site Pages" -Connection  $connectionToSite -Identity  $newPage.PageId -Values @{"FirstPublishedDate" = $pressrelease.published_at; "Created" = $pressrelease.created_at; "Modified" = $pressrelease.modified_at ; "PR_x002d_Categories" = $terms ; "language" = $pressrelease.language; "pressrelease_x002d_url" = $pressrelease.url; "pressroom_name" = $pressrelease.pressroom_name; "Author" = $user.Email; "Editor" = $user.Email }  -UpdateType SystemUpdate`

but as soon as i publish the page it takes my account
` $result=$newPage | Set-PnPPage -Connection  $connectionToSite -Publish `

Basically i want to publish a page from another account like for eg "Martin.Tierson@test.com" , and not from my account
